go/arrow/decimal256/decimal256.go

```diff
@@ -506,7 +506,7 @@ func (n Num) FitsInPrecision(prec int32) bool {
 
 func (n Num) ToString(scale int32) string {
 	f := (&big.Float{}).SetInt(n.BigInt())
-	f.Quo(f, (&big.Float{}).SetInt(scaleMultipliers[scale].BigInt()))
+	f.SetPrec(256).Quo(f, (&big.Float{}).SetInt(scaleMultipliers[scale].BigInt()))
```

Comment:
Is 256 precision always required? Is there a way to know what precision is needed? Or why it doesn't matter?